roles/reset/tasks/main.yml

```diff
@@ -42,6 +42,40 @@
     - /etc/cni
     - /etc/nginx
     - /etc/dnsmasq.d
+    - /etc/dnsmasq.conf
```

Comment:
I typically dislike unrelated changes attached to a PR, but please avoid doing this in the future.